### PR TITLE
QtEmu.QtEmu version 2.1

### PR DESCRIPTION
**Pull request by joknoll**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69166)

### CODE REVIEW

**Comment by knolljo** [comment]
PR was done with `wingetcreate new`, manifest version 1.2.0, the bots error message makes no sense to me.

**Comment by ItzLevvie** [comment]
Packages with `ManifestVersion: 1.2.0` will be accepted in the coming days or so.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing silent installation:
image

File: manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: QtEmu
```
Comment (ItzLevvie) [inline manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.locale.en-US.yaml:7]:
```suggestion
Publisher: QtEmu Developers
```
This must match Control Panel.

File: manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1
```
Comment (ItzLevvie) [inline manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.1.0
```
This must match Control Panel.

File: manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1
```
Comment (ItzLevvie) [inline manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.yaml:5]:
```suggestion
PackageVersion: 2.1.0
```
This must match Control Panel.

File: manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1
```
Comment (ItzLevvie) [inline manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.installer.yaml:5]:
```suggestion
PackageVersion: 2.1.0
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Inactive PR;

**Pull request closed without merging**